src/modules/vectorstores/interfaces/vectorstore.interface.ts

```diff
@@ -1,10 +1,21 @@
 export interface IVectorStoreService {
```

Comment:
The name of the file should be vectorstore.service.interface.ts to match the interface name within.